language-server/src/dotty/tools/languageserver/Main.scala

```diff
@@ -0,0 +1,75 @@
+package dotty.tools
+package languageserver
+
+import java.util.function.Consumer
+
+import java.io.{ File => JFile, InputStream, OutputStream, PrintWriter }
+import java.net._
+import java.nio.channels._
+
+import org.eclipse.lsp4j._
+import org.eclipse.lsp4j.services._
+import org.eclipse.lsp4j.launch._
+
+/** Run the Dotty Language Server.
+ *
+ * This is designed to be started from an editor supporting the Language Server
+ * Protocol, the easiest way to fetch and run this is to use `coursier`:
+ *
+ *   coursier launch $artifact -M dotty.tools.languageserver.Main -- -stdio
```

Comment:
I tried it, and it doesn't do any highlighting on this line, so no point in using this syntax.